build.gradle

```diff
@@ -29,7 +29,7 @@ buildscript {
 
         // 3rd party libs
         cameraKitVersion = '1.0.0-beta3.10'
-        cameraKitJpegKitVersion = '0.1.0'
+        cameraKitJpegKitVersion = '0.2.0'
```

Comment:
I think this will break proguard

Reply:
So this is strange, we need to import this in our main app and `0.2.0` breaks proguard there. Not sure if it will if we bump here.

Reply:
okay I'll revert.